vars/Ubuntu-18.yml

```diff
@@ -0,0 +1,6 @@
+---
+svn_apache_user: www-data
+svn_apache_conf_path: /etc/apache2/sites-enabled
+svn_packages:
+  - subversion
+  - libapache2-mod-svn
```

Comment:
Since this is the _only_ one of Ubuntu vs RedHat variables that are different, I'd rather leave everything as is, but have one more task and one include for `vars/Ubuntu-18.yml` that matches the condition of `Ubuntu 18.04`.

(that way there are only 3 vars files instead of 8).